README.md

```diff
@@ -173,6 +173,94 @@ crud.insert_object('customers', {
 ...
 ```
 
+### Insert many
```

Comment:
You're right, it can be confusing the `batch_*` is used in many places. I have renamed `batch_*` to `*_many_*`  in most cases. New method names were introduced upon request by @Totktonada.